src/swap/utils/processSwapTransaction.ts

```diff
@@ -24,7 +25,9 @@ export async function processSwapTransaction({
   if (approveTransaction?.data) {
     setLifeCycleStatus({
       statusName: 'transactionPending',
-      statusData: null,
+      statusData: {
+        maxSlippage,
+      },
```

Comment:
I was getting build errors in this file post adding the maxSlippage field to lifecycle statusData. I had to update the input params and add the maxSlippage field to all setLifeCycleStatus calls